test/ds/test_bounded_dynamic_ops.py

```diff
@@ -0,0 +1,351 @@
+import os
+import sys
+import unittest
+import torch
+import torch.nn.functional as F
+import torch_xla
+import torch_xla.core.xla_model as xm
+
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+import test_utils
+
+pd = torch._C._EnablePythonDispatcher()
+dev = xm.xla_device()
+
+
+def mark_dynamic(t, dims, bounds):
+  torch_xla._XLAC._xla_mark_bounded_dynamic(t, dims, bounds)
+
+
+def diff_output(testcase,
```

Comment:
Does this function duplicated with TestBoundedDynamicOps::diff_output?

Reply:
Yes, I have removed this function.